utils/proof_parser.py

```diff
@@ -0,0 +1,8 @@
+def parse(obj):
+    # TODO
+    return obj
+
+def encode_data_from_proof_parser(obj):
+    # TODO
```

Comment:
Put more details in these TODO items: what is expected as input, what is to be returned.